runtime/flang/stat_linux.c

```diff
@@ -31,16 +33,20 @@ __fort_setarg(void)
 {
 }
 
+// TODO: Implement functions for WIN32
```

Comment:
Can you file a github issue listing all the TODOs for windows?

Reply:
I don't know what all of the TODOs are for windows. I know that I can use the compiler without this functionality without any issues.